python/cudf/CMakeLists.txt

```diff
@@ -25,7 +25,7 @@ project(
             # language to be enabled here. The test project that is built in scikit-build to verify
             # various linking options for the python library is hardcoded to build with C, so until
             # that is fixed we need to keep C.
-            C CXX
+            C CXX CUDA
```

Comment:
Is this a workaround for the cuCollections fix I made to rapids-cmake? Can it be reverted?

Reply:
I'm not even sure how this snuck in here since I wasn't building Python. Yes, this can definitely be deleted.